docs/basic_analysis.rst

```diff
@@ -75,6 +77,95 @@ And if the spectrum contains a continuum, then it should be subtracted first:
     >>> continuum = Spectrum1D(spectral_axis=spec.spectral_axis, flux=continuum_flux) #doctest:+SKIP
     >>> c = centroid(spec-continuum, region=None) #doctest:+SKIP
 
+Width
+-----
+
+There are several width statistics that are provided by the
+`~specutils.analysis` submodule.
+
+The `~gaussian_sigma_width` function estimates the width of the spectrum by
+computing an approximation of the standard deviation.
+
+The `~gaussian_fwhm` function estimates the width of the spectrum at half max,
+again by computing an approximation of the standard deviation.
+
+Both of these functions assume that spectrum is approximately gaussian and that
+it is centered on the spectral axis.
```

Comment:
This might be a good spot to put an example of using one of these with `SpectralRegion.from_center` to get the centering right?

Reply:
That said... I don't think this has to be that sensitive to the centering (see my comment in the implementation function further below about using `centroid`).  What it *is* sensitive to might be the continuum subtraction, though: I think they assume the continuum has been subtracted.

Reply:
I added an `xfail` unit test for the centered case. Using `centroid` here causes the existing unit tests to fail, so I'm not sure what's going on.